File: javascript-exercicios/ex059.js
/**
 * # crie um programa que leia dois valores e mostre um enu na tela
# [1] somar
# [2] multiplicar
# [3] maior
# [4] novos números
# [5] sair do programa

# seu programa deverá realizar a operação solicitada em casa caso

 */

const prompt = require("prompt-sync")();

let n1 = Number(prompt("digite um valor: "));
let n2 = Number(prompt("digite outro valor: "));
let opcao = 0;
while (opcao != 5){
    console.log(`
        [1] Somar
        [2] Multiplicar
        [3] Maior
        [4] Novos números
        [5] Sair do programa   
    `);
    opcao = Number(prompt(">>>>>>>> opção: "));

    if (opcao == 1){
        let soma = n1 + n2;
        console.log(`Resultado: ${n1} + ${n2} = ${soma}`);
    }
    else if (opcao == 2){
        let multi = n1 * n2;
        console.log(`Resultado: ${n1} x ${n2} = ${multi}`);
    }
    else if (opcao == 3){
        if (n1 > n2){
            var maior = n1;
        }else{
            var maior = n2;
        }
        console.log(`entre ${n1} e ${n2} o maior valor é ${maior}`);
    }
    else if (opcao == 4){
        n1 = Number(prompt("Digite um novo valor: "));
        n2 = Number(prompt("Digite um outro novo valor: "));
    }
    else if (opcao == 5){
        console.log("saindo do programa");
        break
    }
    else{
        console.log("opção inválida! tente novamente");
    }
} 
console.log("fim do programa");

File: curso-python/ex059.py
# crie um programa que leia dois valores e mostre um enu na tela
# [1] somar
# [2] multiplicar
# [3] maior
# [4] novos números
# [5] sair do programa

# seu programa deverá realizar a operação solicitada em casa caso
from time import sleep

num1 = int(input("Digite um valor: "))
num2 = int(input("Digite outro valor: "))

opcao = 0
while opcao != 5:
    print("[1] Somar")
    print("[2] Multiplicar")
    print("[3] Maior")
    print("[4] Novos números")
    print("[5] sair do programa")

    opcao = int(input(">>>>>>>> opção: "))

    if opcao == 1:
        soma = num1 + num2
        print(f"Resultado: {num1} + {num2} = {soma}")
    elif opcao == 2:
        multi = num1 * num2
        print(f"Resultado: {num1} x {num2} = {multi}")
    elif opcao == 3:
        if num1 > num2:
            maior = num1
        else:
            maior = num2
        print("entre {} e {} o maior valor é {}".format(num1,num2,maior))
    elif opcao == 4:
        num1 = int(input("digite um novo valor: "))
        num2 = int(input("digite um outro novo valor: "))
    elif opcao == 5:
        print("saindo do programa")
    else:
        print("opção inválida! tente novamente")
    print("=-="*20)
    sleep(2)
print("FIM DO PROGRAMA")